modules/cnf-performing-end-to-end-tests-running-the-tests.adoc

```diff
@@ -29,31 +24,18 @@ You provide the test image with a `kubeconfig` file in current directory and its
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e FEATURES=performance registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
-/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e LATENCY_TEST_RUNTIME=<time in seconds>\
```

Comment:
For consistency with other work in this PR and also to avoid improper syntax highlighting in the output (it mistakes `time` for a command), make it <time_in_seconds>